bundles/binding/org.openhab.binding.freeswitch/src/main/java/org/openhab/binding/freeswitch/internal/FreeswitchGenericBindingProvider.java

```diff
@@ -74,13 +73,7 @@ public void processBindingConfiguration(String context, Item item, String bindin
 		FreeswitchBindingConfig config = new FreeswitchBindingConfig(item.getName(),item.getClass(), type, argument);
 		
 		addBindingConfig(item, config);
-		
-		Set<Item> items = contextMap.get(context);
-		if (items == null) {
-			items = new HashSet<Item>();
-			contextMap.put(context, items);
-		}
-		items.add(item);
+		super.processBindingConfiguration(context, item, bindingConfig);
 	}
```

Comment:
Removed code duplicated what super.processBindingConfiguration had already done, so moved super call to end.